build/Dockerfile

```diff
@@ -18,12 +18,15 @@ RUN make windows
 # Build promu utility tool, needed to build the windows_exporter.exe metrics binary
 WORKDIR /build/windows-machine-config-operator/promu/
 COPY promu/ .
-RUN go install .
+
+# Create a directory for windows_exporter and set the $GOBIN path for promu installation
+RUN mkdir /build/windows-machine-config-operator/windows_exporter/
```

Comment:
the WORKDIR below(L27) basically does `mkdir` and `cd`. 
So this step can be skipped.

Reply:
`go install .` below should create the directory you're passing as `GOBIN`. Is it facing any permission issues on cachito side?